src/Microsoft.DotNet.Interactive.SqlServer/MsSqlKernel.cs

```diff
@@ -122,5 +99,29 @@ protected override bool CanDeclareVariable(string name, object value, out string
 
             return true;
         }
+
+        protected override void StoreQueryResults(List<TabularDataResource> results, ParseResult commandKernelChooserParseResult)
```

Comment:
Does this need to be `List<TabularDataResource>` as opposed to say `IReadOnlyList<TabularDataResource>`?

Reply:
fixed